examples/falcon_spotlight_vulnerabilities.rs

```diff
@@ -98,7 +92,7 @@ pub struct ApiError(pub String);
 
 impl fmt::Display for ApiError {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        write!(f, "Application Errors: {}", self.to_string())
+        write!(f, "Application Errors: {self}")
```

Comment:
Here we have similar as the one above.
```suggestion
        write!(f, "Application Errors: {}", self.0)
```